obj.js

```diff
@@ -120,7 +120,7 @@ function makeStub(objDescrJSON) {
 function makeFunction(functionName) {
   return function(...args) {
     // this == stub object
-    return callRemote("func", "func-r", {
+    return callRemote("func", {
```

Comment:
What are we doing now, instead of waiting for the result here?

Reply:
`callRemote` always promises a result. The change here is from PR#1 because the argument was unused.